lib/logflare_web/controllers/api/source_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule LogflareWeb.Api.SourceController do
+  use LogflareWeb, :controller
+  alias Logflare.Sources
+  alias Logflare.Users
+
+  def index(%{assigns: %{user: user}} = conn, _) do
```

Comment:
missing fallback controller, but i think you added it in the other PR.

Reply:
yes, tackled in another PR. 

The possible errors are related with lack of auth which would be caught by the plugs, the controller itself can only return lists (either empty or not)